### PR TITLE
DBZ-1818 Distributed tracing blog

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Could you (force-) push this to the _upstream/staging_ branch for preview? Thx!

**Comment by jpechane** [comment]
@gunnarmorling Its there

**Comment by gunnarmorling** [comment]
Hey @jpechane, awesome work -- the actual feature as well as this post! I've added a commit with a few adjustments, mostly to sprinkle some "the"s here and there and to contextualize some things. If you agree with these changes, I'll push this change later today. Thanks a lot, this is really, really cool!

**Comment by gunnarmorling** [comment]
Hey @jpkrohling, if you got some min, your feedback will be appreciated, too. No worries though I think it's in great shape and we can publish it as-as. Thanks :)

File: _posts/2020-12-17-ditributed-tracing.adoc [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+** `db-log-write` with its start timestamp set to database write timestamp.
+The fields from the `source` block are added to the span as *tags*.
+** `debezium-read` with its start time set to the processing timestamp.
+Fields from the envelope are added to the span as *tags*.
+* Optionally, if OpenTracing integration is enabled at the Kafka producer level, a new span is created by the Kafka producer representing the write of the message to a Kafka topic with relevant metadata.
+
+==== Demo
+
+The https://github.com/debezium/debezium-examples/tree/master/outbox[outbox example] was extended with distributed tracing support to demonstrate the functionality.
+This example contains to rudimentary microservices: an order service which exposes a REST API for placing purchase orders, and a shipment service which is notified by the order service about new purchase orders using the outbox pattern.
```
Comment (jpkrohling) [inline _posts/2020-12-17-ditributed-tracing.adoc:89]:
```suggestion
This example contains two rudimentary microservices: an order service which exposes a REST API for placing purchase orders, and a shipment service which is notified by the order service about new purchase orders using the outbox pattern.
```

File: _posts/2020-12-17-ditributed-tracing.adoc [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+And this is where Debezium becomes part of the picture.
+Data change events, as captured by Debezium from a database, and propagated via Kafka Connect and Apache Kafka to one more more downstream consumers are part of a data flow which is very valuable to have insight into.
+How long does it take for change events to flow from source database to sink systems?
+Where is the most time spent in the pipeline?
+Are there any anomalies like spikes in end-to-end lags?
+The integration of distributed tracing with Debezium can help to answer these questions.
+
+== OpenTracing
+
+There are multiple solutions for distributed tracing, but as a starting poin we have decided to follow and use the https://opentracing.io/[OpenTracing] specification.
```
Comment (jpechane) [inline _posts/2020-12-17-ditributed-tracing.adoc:35]:
```suggestion
There are multiple solutions for distributed tracing, but as a starting point we have decided to follow and use the https://opentracing.io/[OpenTracing] specification.
```

**Review by jpkrohling — APPROVED** [review]
There are a couple of typos, but other than that, looks awesome!

**Comment by gunnarmorling** [comment]
> There are a couple of typos, but other than that, looks awesome!

Thanks for the quick check, as always, Juca! Applied all the suggestions, going to merge it.

**Pull request merged**